adapters/AppLovin/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## AppLovin iOS Mediation Adapter Changelog
 
+#### Version 11.8.0.0
+- Verified compatibility with AppLovin SDK 11.8.0.
+- Added support for watermark when bidding.
+- Added support for Medium Rectangle(MREC) ads.
+
 #### Version 11.7.1.0 (In progress)
```

Comment:
```suggestion
#### Version 11.7.1.0
```

Reply:
Can save em some work, since it's "verified" alreaady: https://developers.google.com/admob/ios/mediation/applovin#applovin-ios-mediation-adapter-changelog